vexim/config/functions.php

```diff
@@ -127,6 +127,28 @@ function crypt_password($clear, $salt = '')
                     }
                     $cryptedpass = crypt($clear, $salt);
                 break;
+		case 'sha512':
```

Comment:
I think I have another "ingenious" idea. :) Why don't we allow the users to specify salt prefix instead of hash type? We could still special-case '{SHA}' and 'clear', but that would make the number of algorithms Vexim accepts out-of-the-box completely dependent on what's available on a particular system in a particular time. E.g. for SHA512, the user could to specify `$salt_prefix = '$6$'`, and for BCRYPT – `$salt_prefix = '$2a$10$' (or whatever other number of iterations they want). The `crypt_password()` function would only have to use a salt string long abd clean enough (e.g. 128 characters) and let crypt() do its work.
Thoughts?